packages/cloudflare/src/api/overrides/incremental-cache/kv-incremental-cache.ts

```diff
@@ -23,69 +21,30 @@ class KVIncrementalCache implements IncrementalCache {
     key: string,
     isFetch?: IsFetch
   ): Promise<WithLastModified<CacheValue<IsFetch>> | null> {
-    const cfEnv = getCloudflareContext().env;
-    const kv = cfEnv.NEXT_INC_CACHE_KV;
-    const assets = cfEnv.ASSETS;
-
-    if (!(kv || assets)) {
-      throw new IgnorableError(`No KVNamespace nor Fetcher`);
-    }
+    const kv = getCloudflareContext().env.NEXT_INC_CACHE_KV;
+    if (!kv) throw new IgnorableError("No KV Namespace");
 
-    this.debug(`Get ${key}`);
+    debug(`Get ${key}`);
```

Comment:
I think this should be the other way around i.e. align R2 on what KV was before this PR (see https://github.com/opennextjs/opennextjs-cloudflare/issues/538)